example-ws/src/index.ts

```diff
@@ -17,7 +17,9 @@ const sub = (a: number, b: number): number => {
 wss.on("connection", function connection(ws) {
   console.log("new ws connection");
```

Comment:
it is redundant now

Reply:
this is example, and it can write to console if it wants to imo, no ?

Reply:
yes, but whether you are sure the console is always available ?

Reply:
in this example it is available, i don't think i am supposed to change example use cases here